packages/optimizer/spec/transformers/valid/MinifyHtml/minifies_inline_JSON/expected_output.html

```diff
@@ -1,4 +1,5 @@
- <html ⚡> <head> <script async src="https://cdn.ampproject.org/v0.js"></script> <script async custom-element="amp-img" src="https://cdn.ampproject.org/v0/amp-img-0.1.js"></script> <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;
--moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes-amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes-amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes-amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes-amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style> <noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript> <link href="https://example.com/favicon.ico" rel="icon"> </head> <body>  <amp-analytics type="nielsen"> <script type="application/json">{"vars":{"apid":"XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX","apv":"1.0","apn":"My AMP Website","section":"Entertainment","segA":"Music","segB":"News","segC":"Google AMP"}}</script> </amp-analytics>  <script type="application/ld+json">{"@context":"http://schema.org","@type":"Webpage","url":"http://localhost:8081/documentation/examples/components/amp-bind/ ","name":"amp.dev","headline":"Example: &lt;amp-bind&gt;","description":"amp-bind allows you to add custom interactivity to your pages beyond using AMP&#39;s pre-built components.","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:8080"},"publisher":{"@type":"Organization","name":"AMP Project","logo":{"url":"http://localhost:8080/static/img/icons/icon-512x512.png","width":512,"height":512,"@type":"ImageObject"}}}</script>  <amp-analytics type="nielsen"> <script type="application/json">
+
```

Comment:
Shouldn't blank lines outside of the `<body>` be trimmed as well?

Reply:
Missed that. Fixed that on the parser level (trim doc before parsing).

Reply:
however, spaces after doctype and before `html` still won't be removed.